part-iii-infrastructure-components/event-processing.md

```diff
@@ -12,10 +12,16 @@ The `EventBus` is the mechanism that dispatches events to the subscribed event h
 
 When using the Configuration API, the `SimpleEventBus` is used by default. To configure the `EmbeddedEventStore` instead, you need to supply an implementation of a StorageEngine, which does the actual storage of Events.
 
-```java
+{% codetabs name="Axon Configuration API", type="java" -%}
     Configurer configurer = DefaultConfigurer.defaultConfiguration();
     configurer.configureEmbeddedEventStore(c -> new InMemoryEventStorageEngine());
-```
+{%- language name="Spring Configuration", type="java" -%}
+// somewhere in configuration
+@Bean
+public EventStorageEngine eventStorageEngine() {
```

Comment:
EmbeddedEventStore is default when a StorageEngine is provided. Perhaps we should mention that in the configuration example. Also mention that a JpaEventStorageEngine is automatically created when JPA is detected.